pkg/api/roles.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+)
+
+// API related actions
+const (
+	ActionProvisioningReload = "provisioning:reload"
+)
+
+// API related scopes
+const (
+	ScopeServicesAll           = "services:*"
+	ScopeServicesDashboards    = "services:dashboards"
+	ScopeServicesPlugins       = "services:plugins"
+	ScopeServicesDatasources   = "services:datasources"
+	ScopeServicesNotifications = "services:notifications"
+)
+
+// GetFixedRoleRegistrations returns the list of fixed roles and their grants to organization roles
+// ("Viewer", "Editor", "Admin") or "Grafana Admin" that HTTPServer needs
+func (hs *HTTPServer) GetFixedRoleRegistrations() []accesscontrol.RoleRegistration {
```

Comment:
Shouldn't it be in the provisioning service?

Reply:
Good point, I was hesitant on where it should be defined. I put it there because I think it should be defined in the same package that registers the endpoint. For ENT provisioning, that would have been the same package since it respects the standard package layout. But for OSS provisioning, the API endpoints are defined in the api package and not in the provisioning package.
I'm ok to move it to provisioning, just trying to explain why I set it here. WDYT?

Reply:
Ok had time to think about it. Let's define it here for now. But I agree it needs to be moved to provisioning along with the API endpoints.